sale_order_transmit_method/tests/test_sale_order.py

```diff
@@ -0,0 +1,61 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+
+
+class TestSaleOrder(TransactionCase):
```

Comment:
pls use SavepointCase and move whole setup records to setUpClass

Reply:
Is there any reason for that?

Reply:
very simple: `setUp` runs EVERY time for EVERY test. `setUpClass` only once.
Multiply this for all the tests we run in CI and you get tons of resources and time wasted because of this. Another recommendation is to disable tracking suite wise from setUpClass when not needed:
```py 
cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
Sooner or later I'll add these tips to official OCA guidelines.